main.go

```diff
@@ -60,6 +60,7 @@ func main() {
 		SecretKey:                 cfg.SecretKey,
 		Shutdown:                  cfg.GracefulShutdownTimeout,
 		SignElasticsearchRequests: cfg.SignElasticsearchRequests,
+		HasPrivateEndpoints:	   cfg.HasPrivateEndpoints,
```

Comment:
If you run `go fmt ./...` in dp-search-api it will format this line and others

Also try to keep variables in alphabetical order